dotnet/DotnetModuleSample/DotnetModuleSample/DotnetModuleSample.csproj.user

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
No, this is required for the debug arguments. If this file is excluded the F5 scenario will fail.